src/pages/plugins-2023/index.md.ejs

```diff
@@ -2,7 +2,7 @@
 layout: content.html
 ---
 
-<%- include('plugins') %>
+<%- include('plugins_gen') %>
```

Comment:
plugins_gen here is the generated EJS file created from the previous EJS file and the NPM data by generate_plugins.js - it looks like `<%allPlugins = {...}` on the inside. To get access to this data for the plugins page, it needs to be on the same level as it. Because of this, a plugins_gen.html file is generated by metalsmith - it contains no data but the allPlugins variable. This is why I originally added the html version to the .gitignore. The html file is not referenced on this page

Reply:
Unfortunately that's not possible with the setup as it currently is - either it returns an error that it could not find the include file if the include is anything other than the exact relative/absolute path, or it returns `ENOENT: no such file or directory, open '/home/stokesriona/senecajs/senecajs.org/src/template/partials/plugins_gen.html'`, strangely enough. This is why I had it at the same level as index.md.ejs with the html in the gitigore, the include path for EJS seems to be quite limited